### ISSUE TITLE
Bug: It shouldn't error when local hostname is not resolveable

### ISSUE THREAD

**Issue by samip5** [Testaustime/testaustime-intellij#4]
```
java.net.UnknownHostException: Ryzen-Gaming-Gentoo: Ryzen-Gaming-Gentoo: Name or service not known
	at java.base/java.net.InetAddress.getLocalHost(InetAddress.java:1657)
	at fi.testaustime.plugin_intellij.services.TestaustimeApplicationService.pingNow(TestaustimeApplicationService.kt:73)
	at fi.testaustime.plugin_intellij.services.TestaustimeApplicationService.startScheduledPinger$lambda-0(TestaustimeApplicationService.kt:60)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
	at com.intellij.util.concurrency.SchedulingWrapper$MyScheduledFutureTask.run(SchedulingWrapper.java:275)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:702)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:699)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:699)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.net.UnknownHostException: Ryzen-Gaming-Gentoo: Name or service not known
	at java.base/java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
	at java.base/java.net.InetAddress$PlatformNameService.lookupAllHostAddr(InetAddress.java:933)
	at java.base/java.net.InetAddress.getAddressesFromNameService(InetAddress.java:1529)
	at java.base/java.net.InetAddress$NameServiceAddresses.get(InetAddress.java:852)
	at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1519)
	at java.base/java.net.InetAddress.getLocalHost(InetAddress.java:1652)
	... 12 more
```

**Comment by bluelhf** [comment]
Hm. We really want the [_computer name_](https://stackoverflow.com/a/40702767) here, not the hostname (which might not resolve, or there might be multiple, or it might not exist).

The standard library has no way to obtain this information, so we'd have to resort to a system-specific library like [gethostname4j](https://github.com/mattsheppard/gethostname4j).

The [OpenJDK bug for this](https://bugs.openjdk.org/browse/JDK-8169296) has been open since 2016, and awaits a response from Alan Bateman.

For now, you can work around the issue by adding your computer name to `/etc/hosts` such that it resolves to the loopback address:
```
127.0.0.1 Ryzen-Gaming-Gentoo
```

**Comment by samip5** [comment]
What is the computer name used for?
I think it's not exposed in UI at least on Testaustime frontend.

**Comment by developerfromjokela** [comment]
> What is the computer name used for?
> I think it's not exposed in UI at least on Testaustime frontend.

Honestly, no idea. We only implemented testaustime API using documentation available. Backend guys maybe can explain the reason. It may be saved for future use case. 😉

**Comment by Chicken** [comment]
Probably implemented in API as it's something WakaTime does as well. It's a nice metric to known which machines you use if you use multiple ones. Should probably be implemented in the frontend. How weird that Java does not have a function to get the "computer name". Just use the gethostname4j library I guess?

**Comment by samip5** [comment]
But yes, that issue went away by just adding the hostname to /etc/hosts. :)

**Comment by bluelhf** [comment]
Should be fixed by c0308b84dce1d854b10f9e201104682ff666bcc7
